Doc/library/datetime.rst

```diff
@@ -589,8 +589,8 @@ Supported operations:
 +-------------------------------+----------------------------------------------+
 | Operation                     | Result                                       |
 +===============================+==============================================+
-| ``date2 = date1 + timedelta`` | *date2* is ``timedelta.days`` days removed   |
-|                               | from *date1*. (1)                            |
+| ``date2 = date1 + timedelta`` | *date1* is ``timedelta.days`` days after     |
+|                               |  *date2*. (1)                                |
```

Comment:
Changing "removed from" to "after" is good, but swapping `date1` with `date2` in only one column is wrong.  `+` here does an obvious thing, not a bizarre thing :wink:.